### ISSUE TITLE
Issue w/ Custom Tabs & Fenced Code Blocks

### ISSUE THREAD

**Issue by kahboom** [redhat-ipaas/redhat-ipaas.github.io#15]
Custom tabs are working, but only with blockquotes and not with fenced code blocks.

Ref: https://github.com/middleman/middleman-syntax

**Comment by kahboom** [comment]
For more context, there are two plugins - the default `Multilang` plugin located in `lib/multilang.rb`, and the custom `Tabs` plugin located in `lib/tabs.rb`, which are nearly identical. To prevent conflict, I've replaced the use of the Multilang with Tabs for one of the layouts (`source/layouts/developers.erb` and it's main index file at `source/developers/index.html.md`), since I figured it would provide the same functionality.

Normally, the way the tab/code syntax feature for Slate works is that you'd specify the languages at the top:

```
language_tabs:
  - ruby: Design
```

And can specify code to be displayed only for the 'Ruby' tab (in this case it is labeled as 'Design' because Slate allows this) like so:

```
```ruby
puts 'Hello world!'
``
```

You can also do:

```
<blockquote class="language-specific ruby">
You must replace <code>meowmeowmeow</code> with your personal API key.
</blockquote>
```

If you specify a language that is not defined at the top, it will always show (see [here](https://github.com/lord/slate/wiki/Customizing-the-Language-Tabs)). All of this works, but when I replace it with my "custom" tabs plugin, which is essentially a copy and paste of the language tabs, and renaming, only the conditional `<blockquote>` seems to work, not the conditional fenced code block.

For instance, the following works:

```
---
title: Developers

area_tabs:
  - design: Design
  - backend: Backend

search: true
---

<blockquote class="area-specific design">
You must replace <code>meowmeowmeow</code> with your personal API key.
<img src="/path/to/image.jpg">
</blockquote>
```

It only shows the `<blockquote>` when on the `design` tab. However, the code fence conditioning doesn't seem to work anymore:

```
```backend
Backend
This will show up on every tab, regardless of the fact that I've declared it at the top of the file.
``
```

Perhaps it's best to just use the Multilang tab and deal with having to map things like in the first example, using blockquotes and fenced code blocks to get things to display for each tab. It just feels a bit hack-y to me. Plus I can never remember what `ruby` or `c` maps to further down in the file and keep having to scroll all the way back up.

EDIT: Plus I also want to be able to use syntax highlighting when I need it, specific to each language.
